docs/reference/cat/indices.asciidoc

```diff
@@ -50,6 +50,10 @@ indexing and search. As a result, all document counts include hidden
 To get an accurate count of {es} documents, use the <<cat-count,cat count>> or
 <<search-count,count>> APIs.
 
+Note that information such as document count, deleted document count and store size are not shown for
+<<snapshots-source-only-repository, source-only indices>> since these indices do not contain the relevant
```

Comment:
The term `source-only indices` is not used elsewhere and may be confusing. Could we call them "indices restored from a source-only snapshot" instead just to avoid any ambiguity?